src/Cake.Core.Tests/Unit/IO/NuGet/PackageReferenceBundlerTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Cake.Core.IO;
+using Cake.Core.IO.NuGet;
+using NSubstitute;
+using Xunit;
+
+namespace Cake.Core.Tests.Unit.IO.NuGet
+{
+    public sealed class PackageReferenceBundlerTests
+    {
+        public sealed class TheParseMethod
```

Comment:
I'll give this a try -- some of the fixtures tend to have the opposite effect though (sometimes hiding meaningful details in another file).